### DISCUSSION TITLE
[Bug report] Theme "stuck" in store - cannot purchase Changelog theme

### DISCUSSION THREAD

**Discussion by MarcoBhoff** [GetPublii/Publii#1554] [Bug Report]
### Operating system

Windwos 10

### Publii version

None - theme store problem

### Issue type

Something else

### Bug description

When in the theme store, and on the Changelog page - https://marketplace.getpublii.com/themes/changelog/ - there is 2 items in the cart:
![image](https://github.com/GetPublii/Publii/assets/9840195/726cf99a-7ca8-409d-beed-01cccf008440)

When I click on "Proceed to checkout" or the cart, it is empty:
![image](https://github.com/GetPublii/Publii/assets/9840195/07dff1bb-dafb-4d15-8723-c4b28271bdf1)
![image](https://github.com/GetPublii/Publii/assets/9840195/78b7e584-a7c5-48d1-b303-6b14435dc1c5)

I have tried different browsers and incognito mode. A friend tried as well, an he is facing the same problem. 2 items for €78 in the cart when on the Changelog theme, but none when going to cart - he has not added anything to the card, I added the theme myself.

If going to another theme there is nothing in the cart either:
![image](https://github.com/GetPublii/Publii/assets/9840195/ba69c58a-51d4-4fc0-8593-875ddfc50d20)

If adding another theme to the cart, and I go to checkout, that is the only thing in the cart:
![image](https://github.com/GetPublii/Publii/assets/9840195/83004028-b854-4cbc-900b-46afd8f60126)
![image](https://github.com/GetPublii/Publii/assets/9840195/c8f716c2-3f3b-44e0-a6c2-c2d87bc473ab)

### Steps to reproduce

Written in description

**Comment by VilgotS** [comment]
I have the same issue

**Comment by bobmitro** [comment]
Please try clearing the browser cache. I have checked it in various browsers, including incognito mode, and I don't see the issue you're describing. The shopping cart is empty; I honestly don't see any reason why it should be full by default.

**Reply by MarcoBhoff** [reply]
Yeah, I thought it was very weird. I tried clearing cache and different browsers. I believe @VilgotS didn't even add anything to the cart and still saw 2 items. I added the Changelog theme myself, and therefore understand why it's there for me, but not why they aren't there when I go to the cart.

I have never done web dev but it smells kind of a cache problem to me since me and Vilgot both got the problem.

**Reply by MarcoBhoff** [reply]
Here is a gif, now I only have 1 item in the cart however

https://github.com/GetPublii/Publii/assets/9840195/382a96bd-c4e6-442f-bc3b-c4176105a4bd

**Comment by bobmitro** [comment]
I cannot replicate a similar case, so I'm not sure what the issue might be. One possibility is that cookies might be disabled in your browser settings, preventing the cart from updating correctly. However, since the cart works with other themes, it seems cookies are enabled. No one else has reported a similar issue, and the theme sells without problems.

**Reply by MarcoBhoff** [reply]
I understand, I will try again later and hope it works.
